frontend/src/views/ChatView.vue

```diff
@@ -356,20 +370,12 @@ function generatePromptWithDebug() {
           />
           <PgButton
             severity="info"
-            :icon="detailsVisible ? 'pi pi-times' : 'pi pi-info'"
+            icon="pi pi-info"
             :disabled="!selectedDictionary"
             class="h-fit m-2 ml-0"
-            :aria-label="
-              detailsVisible
-                ? t('chat.dictionary.details.hide_details')
-                : t('chat.dictionary.details.show_details')
-            "
-            :title="
-              detailsVisible
-                ? t('chat.dictionary.details.hide_details')
-                : t('chat.dictionary.details.show_details')
-            "
-            @click="toggleDetails"
+            title="t('chat.dictionary.details.show_details')"
+            aria-label="t('chat.dictionary.details.show_details')"
```

Comment:
```suggestion
            :title="t('chat.dictionary.details.show_details')"
            :aria-label="t('chat.dictionary.details.show_details')"
```
senza i : tratta tutto quello tra apici come stringhe invece di interpretarlo